README.md

```diff
@@ -1,6 +1,5 @@
 [![sfv4_00000](https://github.com/serverless/serverless/assets/2752551/6cf62477-7834-49dc-9a41-f71b76d70a18)](https://github.com/serverless/serverless/tree/v4.0)
 
-
```

Comment:
`npm run prettier-check` fails with this extra newline, not sure how it got committed but didn't want the CI to fail for this PR so ran `npm run prettify` which fixed it by removing this newline.